app/logic/Calendar/Event.ts

```diff
@@ -194,6 +198,29 @@ export class Event extends Observable {
     this.response = original.response;
   }
 
+  get outgoingInvitation() {
+    return new OutgoingInvitation(this);
+  }
+
+  protected participantMe(): { identity: MailIdentity, participant: Participant, person: PersonUID } {
+    let results = [];
+    for (let account of appGlobal.emailAccounts) {
+      if (!account.canSendInvitations) {
+        continue;
+      }
+      for (let identity of account.identities) {
```

Comment:
Use `findIdentityForEMailAddress()`

Reply:
That searches accounts which I can't use.

Reply:
Here's what I would do:
* Given that you need it only for outgoing invitations, move the function to the `OutgoingInvitation` class
* Then you can search for the participant which is the organizer. And you can rename it to `organizerParticipant()`
* Validate this result by check that the email address of this participant belongs to our user. You can use `findIdentityForEMailAddress()` for that. It's only a validation, a double-check, to ensure that we won't take over a meeting organized by somebody else. The check could also be elsewhere.